File: sdk/dart/lib/model/shipment.dart
part of swagger.api;


@Entity()
class Shipment {
  
  @Property(name: 'ID')
  String ID = null;
  

  @Property(name: 'Shipper')
  String shipper = null;
  

  @Property(name: 'DateShipped')
  DateTime dateShipped = null;
  

  @Property(name: 'TrackingNumber')
  String trackingNumber = null;
  

  @Property(name: 'Cost')
  Number cost = null;
  

  @Property(name: 'Items')
  List<ShipmentItem> items = [];
  

  @Property(name: 'xp')
  Object xp = null;
  
  Shipment();

  @override
  String toString()  {
    return 'Shipment[ID=$ID, shipper=$shipper, dateShipped=$dateShipped, trackingNumber=$trackingNumber, cost=$cost, items=$items, xp=$xp, ]';
  }

}

File: sdk/javascript-closure-angular/API/Client/Shipment.js
goog.provide('API.Client.Shipment');

/**
 * @record
 */
API.Client.Shipment = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.Shipment.prototype.ID;

/**
 * @type {!string}
 * @export
 */
API.Client.Shipment.prototype.shipper;

/**
 * @type {!API.Client.date}
 * @export
 */
API.Client.Shipment.prototype.dateShipped;

/**
 * @type {!string}
 * @export
 */
API.Client.Shipment.prototype.trackingNumber;

/**
 * @type {!number}
 * @export
 */
API.Client.Shipment.prototype.cost;

/**
 * @type {!Array<!API.Client.ShipmentItem>}
 * @export
 */
API.Client.Shipment.prototype.items;

/**
 * @type {!API.Client.Object}
 * @export
 */
API.Client.Shipment.prototype.xp;

